app/Services/Campaign/ExportService.php

```diff
@@ -46,13 +48,28 @@ public function queue(): self
         $this->campaign->export_date = date('Y-m-d');
         $this->campaign->saveQuietly();
 
-        Export::dispatch($this->campaign, $this->user, false);
-        Export::dispatch($this->campaign, $this->user, true);
+        $entitiesExport = CampaignExport::create([
+            'campaign_id' => $this->campaign->id,
+            'created_by' => $this->user->id,
+            'type' => CampaignExport::TYPE_ENTITIES,
+            'status' => CampaignExport::STATUS_SCHEDULED,
+        ]);
+
+        Export::dispatch($this->campaign, $this->user, $entitiesExport, false);
+
+        $assetExport = CampaignExport::create([
+            'campaign_id' => $this->campaign->id,
+            'created_by' => $this->user->id,
+            'type' => CampaignExport::TYPE_ASSETS,
+            'status' => CampaignExport::STATUS_SCHEDULED,
+        ]);
+
+        Export::dispatch($this->campaign, $this->user, $assetExport, true);
 
         return $this;
     }
 
-    public function export(): self
+    public function export(): int
```

Comment:
export() returning an int smell of base code. Make a new function size() to get the size of the export.